### PR TITLE
[bindings] Test SimbodyMatterSubsystem::calcResidualForce()

### PR DESCRIPTION
**Pull request by chrisdembia**
This PR just makes sure that the method SimbodyMatterSubsystem::calcResidualForce() and the class OpenSim::InverseDynamicsSolver are accessible from MATLAB/Python.

### CODE REVIEW

**Comment by tkuchida** [comment]
LGTM :+1:

**Comment by chrisdembia** [comment]
Thanks Tom; anyone interested in reviewing?

**Comment by aymanhab** [comment]
@chrisdembia This looks great in terms of verifying the right sizes of outputs and that the methods exist, how hard is it to check for actual meaningful values?

**Comment by tkuchida** [comment]
> check for actual meaningful return values

Good idea. I don't think a regression test is appropriate here, but it would be nice to ensure all the values aren't `0` or `NaN`, for example.

**Comment by chrisdembia** [comment]
I have added more rigorous checks in the tests, and I also added to the CHANGELOG that the SimbodyMatterSubsystem is now available in scripting.

Ready for review again.

**Comment by humphreysb** [comment]
@chrisdembia - I will look at it over the weekend.

**Comment by chrisdembia** [comment]
Great; thank you.

File: CHANGELOG.md [hunk 1]
```diff
@@ -79,8 +79,14 @@ Removed Classes
 The following classes are no longer supported in OpenSim and are removed in OpenSim 4.0.
 - Muscle class `ContDerivMuscle_Depredated`.
 
-Python
-------
+MATLAB and Python interfaces
+----------------------------
+- The SimbodyMatterSubsystem class--which provides operators related to the mass
+matrix, jacobians, inverse dynamics, etc.--is now accessible in MATLAB and
```
Comment (tkuchida) [inline CHANGELOG.md]: jacobians -> Jacobians (named after Carl Gustav Jacob Jacobi).

**Comment by tkuchida** [comment]
One minor comment; otherwise, looks good to go :+1:

File: Bindings/Java/Matlab/tests/testSimbody.m [hunk 2]
```diff
@@ -0,0 +1,40 @@
+
+import org.opensim.modeling.*
+
+% Create a model.
+model = Model('arm26.osim');
+
+s = model.initSystem();
+
+model.realizeDynamics(s);
```
Comment (humphreysb) [inline Bindings/Java/Matlab/tests/testSimbody.m]: Looks Good.  One question, aren't you only setting acceleration for one of the coordinates?

**Comment by aymanhab** [comment]
After addressing @humphreysb clarification (in code comments) this should be ready to merge, as a significant improvement over what we have. Thanks @chrisdembia

**Comment by chrisdembia** [comment]
Thanks @tkuchida, @aymanhab and @humphreysb for comments. Based on @aymanhab's permission, I am merging this.

**Pull request merged**